### PR TITLE
fix(zone.js): check if `process` is defined when patching the `GlobalErrors.install`

### PR DESCRIPTION
**Pull request by arturovt**
## PR Checklist
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type
- [x] Bugfix

## What is the current behavior?
Issue Number: #42260

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/zone.js/lib/jasmine/jasmine.ts [hunk 1]
```diff
@@ -66,13 +66,28 @@ Zone.__load_patch('jasmine', (global: any, Zone: ZoneType, api: _ZonePrivate) =>
         if (originalInstall && !instance[symbol('install')]) {
           instance[symbol('install')] = originalInstall;
           instance.install = function() {
-            const originalHandlers = process.listeners('unhandledRejection');
-            const r = originalInstall.apply(this, arguments);
-            process.removeAllListeners('unhandledRejection');
+            const isNodeJs = typeof process !== 'undefined' && !!process.on;
+            // Note: Jasmine checks internally if `process` and `process.on` is defined. Otherwise,
+            // it installs the browser rejection handler through `global.global.addEventListener`.
```
Comment (JiaLiPassion) [inline packages/zone.js/lib/jasmine/jasmine.ts:71]: should be `global.addEventListener`?

Comment (arturovt) [reply packages/zone.js/lib/jasmine/jasmine.ts:71]: Yeah, typo.

**Review by JiaLiPassion — APPROVED** [review]
LGTM! thank you for the PR.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit c7bcc1b50182e5378756aa4528a24075b5be026e.

**Pull request closed without merging**